modules/data_integrity_flag/templates/menu_data_integrity_flag.tpl

```diff
@@ -0,0 +1,4 @@
+{*
+  Everything is rendered in index.js now!
+  I'm here because currently Loris breaks without me :(
+*}
```

Comment:
I know @drusian does like when this is delete but can't you do something similar to what you did in Media?